Moved docs media to dedicated folders

valerioangelici opened the pull request:
I just moved the media images for the assembly guide and print guide to dedicated folders so it's cleaner and easier to know where image is used.
I Updated the links in the md files too, but being them /RootCNC/Root_4_Lite/main they will not show up until the pr is merged to the main branch.

Commit:
moved print info media to their own folder

Commit:
Merge branch 'wiki-print-info-updates' of github.com:valerioangelici/Root_4_Lite into wiki-print-info-updates

Commit:
moved assembly guide media to their own folder

RootCNC commented:
Ah amazing, I did wonder if you spotted the links in the files being an absolute directory link, rather than a relative one.

The reason for this, is due to the individual repos (Root 4, Root 4 Lite, Root Controller etc) get automatically collated together into the Wiki Repo. from then its distributed to the [main website](https://rootcnc.com/) and then the [wiki site](https://wiki.rootcnc.com/). 

I'll site down with some fresh eyes and review all the changes - but on first inspection, looks great. thank you.

valerioangelici commented:
Great!
I just moved the images and updated the paths in the files.
Content updates will eventually come after this is merged so we don't mix
reorganization with content updates.
I am taking screenshots on the slicer while I print the parts to document
when and where supports are needed so it's clearer.

Il mer 14 set 2022, 23:43 Pete ***@***.***> ha scritto:

> Ah amazing, I did wonder if you spotted the links in the files being an
> absolute directory link, rather than a relative one.
>
> The reason for this, is due to the individual repos (Root 4, Root 4 Lite,
> Root Controller etc) get automatically collated together into the Wiki
> Repo. from then its distributed to the main website <https://rootcnc.com/>
> and then the wiki site <https://wiki.rootcnc.com/>.
>
> I'll site down with some fresh eyes and review all the changes - but on
> first inspection, looks great. thank you.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/RootCNC/Root_4_Lite/pull/3#issuecomment-1247331990>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AALY5B6F4AGODNEP3C534NLV6JBIBANCNFSM6AAAAAAQMGE7NI>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>

RootCNC commented:
Spot on - I'll adopt this structure going forward :) 

the Wiki and website should update in due time.

Thank you
